test/e2e/problemmaker/problem_maker.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/spf13/pflag"
+
+	"k8s.io/node-problem-detector/test/e2e/problemmaker/makers"
+)
+
+func init() {
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+}
+
+type options struct {
+	// Command line options. See flag descriptions for the description
+	Rate    float32
+	Timeout time.Duration
+	Problem string
+}
+
+// AddFlags adds log counter command line options to pflag.
+func (o *options) AddFlags(fs *pflag.FlagSet) {
+	fs.Float32Var(&o.Rate, "rate", 1.0,
```

Comment:
I was planning to keep it as a float, so that we allow rate such as "100 problems per minute". If we only allow int, then we could only have "60 problems per minute" or "120 problems per minute".
This is used to stress NPD and measure its CPU usage under heavy log processing. I'm not sure whether "1 per second" will be a good granularity.
I don't have any strong opinion through.

Reply:
I am confused. The description of this flag is "Number of times the problem should be generated **per second**". Even if use int here, you can still have 100 problems **per minute**, right?

Reply:
"100 problems per minute" is actually "1.667 problems per second" :) which cannot be expresses as using an `int`

Reply:
Oh, I see what you mean now.